src/databricks/labs/remorph/snow/databricks.py

```diff
@@ -621,3 +621,9 @@ def update_sql(self, expression: exp.Update) -> str:
                 sql = f"UPDATE {this} SET {set_sql}{expression_sql}{order}{limit}"
 
             return self.prepend_ctes(expression, sql)
+
+    def generate(self, expression: exp.Expression, copy: bool = True, **opts) -> str:
```

Comment:
this should be re-written as `transforms`
https://sqlglot.com/sqlglot/transforms.html